packages/xo-web/src/xo-app/settings/remotes/remote.js

```diff
@@ -284,7 +284,7 @@ export default decorate([
                     messages.remoteSmbPlaceHolderPassword
                   )}
                   required
-                  type='text'
+                  type='password'
                   value={password}
                 />
```

Comment:
@pdonias Do you think we should use our `Password` component which allows displaying the password?